src/timer.rs

```diff
@@ -16,7 +16,7 @@ impl Timer {
             counter: 0,
             modulo: 0,
             enabled: false,
-            step: 256,
+            step: 1024, //256 * 4
```

Comment:
the step at the beginning is 256 machine cycles so 1024 clock tick.
and you work with clock cycles for div step, you give to this function clock tick, so step at the beginning need to be 1024 clock tick